src/app/frontend/_variables.scss

```diff
@@ -50,3 +50,4 @@ $bold-font-weight: 500;
 $layout-breakpoint-lg: 1280px;
 $whiteframe-shadow-1dp: 0 1px 3px 0 rgba(0, 0, 0, .2), 0 1px 1px 0 rgba(0, 0, 0, .14), 0 2px 1px -1px rgba(0, 0, 0, .12);
 $baseline-grid: 8px;
+$bar-height: 2px;
```

Comment:
Move $bar-height: 2px; to deployfromsettings.scss. This variable is used only there.